substra/sdk/client.py

```diff
@@ -721,6 +721,54 @@ def download_objective(self, key: str, destination_folder: str) -> None:
             os.path.join(destination_folder, 'metrics.py'),
         )
 
+    @logit
+    def download_model(self, key: str, folder) -> None:
```

Comment:
Maybe make this function private ? The user will probably want to use `download_traintuple_model` or another, and may be confused by `download_model`.

Completely subjective, ignore if you want

Reply:
It's actually intentional to make this function public to allow model downloads by model key. For instance in `interface.py`: https://github.com/SubstraFoundation/substra/pull/261/files#diff-08e9c84da66e25e024f4353cfbf90c4572f99d08219ded2b03dded7a0c6b1106R1088

Reply:
I did not think we would allow this actually, missed it in the spec, please ignore this comment then.